src/Chapter1.hs

```diff
@@ -568,7 +573,10 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel c 
+    | c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u' = True
```

Comment:
Absolutely correct!  👍🏻 

As `||` alread returns a value of type `Bool`, you don't need to use guards and return the result of operator application directly:

```haskell
isVowel c =
        c == 'a' || c == 'A'
    ||  c == 'e' || c == 'E'
    || ...
```